src/zoid/venmo/component.jsx

```diff
@@ -57,10 +57,9 @@ export function getVenmoCheckoutComponent(): VenmoCheckoutComponent {
 
       defaultContext: supportsPopups() ? CONTEXT.POPUP : CONTEXT.IFRAME,
 
+      // $FlowIssue problem with the multiple types this prop takes
       domain: [getPayPalDomainRegex(), getVenmoDomainRegex()],
 
-      logger: getLogger(),
```

Comment:
I feel comfortable removing this `logger` because this component is brand new and not being used in production